### PR TITLE
Validation: Removed Duplicate Headers and Added Missing Alt Text

### PR DESCRIPTION
**Pull request by dereklh7**
Removed duplicate headers and added missing alt text.

### CODE REVIEW

File: aspnet/web-forms/overview/moving-to-aspnet-20/profiles-themes-and-web-parts.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 uid: web-forms/overview/moving-to-aspnet-20/profiles-themes-and-web-parts
 title: "Profiles, Themes, and Web Parts | Microsoft Docs"
 author: rick-anderson
-description: "There are major changes in configuration and instrumentation in ASP.NET 2.0. The new ASP.NET configuration API allows for configuration changes to be made pr..."
+description: Describes the profiles, themes, and web parts for ASP.NET 2.0 and provides overviews and tables for each category.
```
Comment (kturetz) [inline aspnet/web-forms/overview/moving-to-aspnet-20/profiles-themes-and-web-parts.md:5]: "A S P dot Net 2 point zero."

Hey, is this a test to see if I'm paying attention?

Comment (dereklh7) [reply aspnet/web-forms/overview/moving-to-aspnet-20/profiles-themes-and-web-parts.md:5]: Not a test - we don't need to write this for meta descriptions.

File: aspnet/web-forms/overview/moving-to-aspnet-20/server-controls.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ In ASP.NET 2.0, control state solves that problem nicely via the introduction of
 
 This video is a walkthrough of view state and control state.
 
-![](server-controls/_static/image1.png)
+![Screenshot of the walkthrough video that describes the view state and control state fields, showing a Windows Explorer browser window.](server-controls/_static/image1.png)
```
Comment (kturetz) [inline aspnet/web-forms/overview/moving-to-aspnet-20/server-controls.md:57]: "walk through"

File: aspnet/web-forms/overview/moving-to-aspnet-20/server-controls.md [hunk 3]
```diff
@@ -190,7 +190,7 @@ It is fairly common to encounter ASP.NET developers struggling with attempting t
 
 This video is a walkthrough of the Wizard control.
 
-![](server-controls/_static/image2.png)
+![Screenshot of a video walkthrough of the Wizard Control with the Server Controls screen with a Microsoft Visual Studio window is displayed.](server-controls/_static/image2.png)
```
Comment (kturetz) [inline aspnet/web-forms/overview/moving-to-aspnet-20/server-controls.md:193]: "Screenshot of a video walkthrough of the Wizard Control. The Server Controls screen with a Microsoft Visual Studio window is displayed."

**Comment by dereklh7** [comment]
@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**